app/policies/application_policy.rb

```diff
@@ -7,21 +7,25 @@ def initialize(user, record)
   end
 
   def index?
-    false
+    true
   end
 
   def show?
-    false
+    true
   end
 
   def create?
-    false
+    true
   end
 
   def new?
     create?
   end
 
+  def check?
```

Comment:
If this is the same as `new?` you should use `alias_method`.
https://github.com/varvet/pundit?tab=readme-ov-file#plain-old-ruby